modules/gapi/src/backends/onnx/gonnxbackend.cpp

```diff
@@ -165,14 +165,18 @@ inline int getIdxByName(const std::vector<cv::gimpl::onnx::TensorInfo>& info, co
     return std::distance(info.begin(), it);
 }
 
-inline int toCV(ONNXTensorElementDataType prec) {
+std::tuple<int, bool> toCV(ONNXTensorElementDataType prec) {
```

Comment:
I think you can understand function if you look at code (Warning especially).

Reply:
Obviously, you can, but it will be like an assumption indirectly supported by a warning. Comment would be just for clearance for the future generations :)
Anyway, as you wish